Add view for asking questions to admin users

MathiasLantean opened the pull request:
This API is not related to any model. The question is just sent by email to all the admin users in the system.

Commit:
Add view for asking questions to admin users

This API is not related to any model. The question is just sent by email to all the admin users in the system.

kaozdl commented on contact/views.py:
```diff
@@ -1,10 +1,33 @@
+        serializer = QuestionSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        data = serializer.validated_data
+        admin_users = User.objects.filter(is_superuser=True).values_list('email', flat=True)
```
```suggestion
        admin_users = User.objects.filter(is_superuser=True, is_active=True).values_list('email', flat=True)
```

kaozdl reviewed (commented):
Looks Good!

Commit:
Code refactor

Use ViewSet instead of APIView for cleaning URLs code.
Add is_active filter while filtering admin users for sending emails.

guilleijo reviewed (approved):
👍
